src/OrchardCore.Modules/OrchardCore.Redis/Startup.cs

```diff
@@ -31,13 +34,26 @@ public Startup(ShellSettings shellSettings, IShellConfiguration configuration, I
             _logger = logger;
         }
 
+        private static bool AlwaysValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
```

Comment:
Move the private method underneath, also you could use body expression

Reply:
Done.